mcmd/commands/add.py

```diff
@@ -72,6 +72,29 @@ def arguments(subparsers):
                              type=str,
                              help="The token")
 
+    p_add_theme = p_add_subparsers.add_parser('theme',
+                                              help='Upload a bootstrap theme')
+    p_add_theme.set_defaults(func=add_theme,
+                             write_to_history=True)
+    p_add_theme.add_argument('bs3',
+                             type=str,
+                             metavar='Bootstrap 3 theme',
+                             help="The bootstrap 3 css theme file")
+    p_add_theme.add_argument('--bs4',
```

Comment:
Same here, but then you could also include a short option:
`'--bootstrap4', '-bs4'`